client/src/lib/dependency-injection/graphql/buildCreateMutation.js

```diff
@@ -1,13 +1,9 @@
 import { defaultTag } from './tags';
-import { getSingularName, getFields, getFragments } from './helpers';
+import { getSingularName, getFields, getFragments, getMutationParams, getMutationVariables } from './helpers';
 
 const buildCreateMutation = (tag = defaultTag) => (
-  tag`mutation Create${getSingularName}(
-    $Input:${getSingularName}CreateInputType!
-  ) {
-    create${getSingularName}(
-      Input: $Input
-    ) {
+  tag`mutation Create${getSingularName}${getMutationVariables} {
+    create${getSingularName}${getMutationParams} {
       ${getFields}
     }
   }
```

Comment:
Turning `$Input:${getSingularName}CreateInputType!` and `Input: $Input` into default fallbacks, which can be overridden by setting explicit params. See the new related test for an example.